components/ongage/actions/find-subscriber/find-subscriber.js

```diff
@@ -0,0 +1,29 @@
+const ongage = require('../../ongage.app.js');
+
+module.exports = {
+  key: 'ongage-find-subscriber',
+  name: 'Ongage Find Subscriber',
+  description: 'Find a list subscriber in Ongage.',
+  version: '0.0.2',
```

Comment:
Since you're committing the files for the first time to the global Pipedream registry, can you modify these versions to `0.0.1`? That's typically the version we set for new components, and then apply semver after (e.g. bump patch version for bug fixes, bump minor for new features, etc.)